spa/package.json

```diff
@@ -9,13 +9,14 @@
     "generate:graphql": "graphql-codegen"
   },
   "dependencies": {
-    "@apollo/client": "0.0.0-pr-10830-20230502215140",
+    "@apollo/client": "0.0.0-pr-10830-20230608162519",
```

Comment:
I'm using a snapshot release here because this PR's `MultipartBatchHttpLink` uses a feature from the unreleased 3.8 beta **and** the fix for the variables issue has not landed in the beta yet - I'm opening the PR for that today.